Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -1420,7 +1420,12 @@ where packageDirs.Contains(path)
             }
             try {
                 string pathInZip = entry.FullName.Replace('/', '\\');
-                string path = Path.Combine(zipFileName, pathInZip);
+                string path;
+                try {
+                    path = Path.Combine(zipFileName, pathInZip);

```

Comment:
The CommonUtils equivalent here is better (if it works) - we'll be able to exceed 260 chars, which I believe actually works okay here.